### DISCUSSION TITLE
Allow us to set a custom image for "Desktop"

### DISCUSSION THREAD

**Discussion by Nandru86** [LizardByte/community#35] [Sunshine]
Now that we can set poster images for our apps, the desktop's default poster looks... unappealing. It would be nice if we can set a poster for it as well

**Comment by lucksufe** [comment]
you can replace the default image in sunshine dir

**Comment by acarbonaro** [comment]
Agreed; hard-coded options don't work for everyone.  It'd be nice to have some customization here.

[a comment by an automated account is not shown]
